plugins/wasi_http/func.cpp

```diff
@@ -13,21 +13,18 @@
 namespace WasmEdge {
 namespace Host {
 
-Expect<void> WasiHttpPrint::body(const Runtime::CallingFrame &,
```

Comment:
Do the components not need the calling frame information?

Reply:
Yes, because component cannot directly touch any CallingFrame, but indirectly via original module.